webpack.config.production.js

```diff
@@ -33,6 +33,9 @@ module.exports = {
               sourceMap: true,
```

Comment:
This file is missing the `test: /\.(css|scss)$/`. Implement the change you have on line 22 of the development config

Reply:
Alright. Thanks for the observation